integration/app/test-demo/webpack.config.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path');
+const InlineChunkHtmlPlugin = require('react-dev-utils/InlineChunkHtmlPlugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const app = 'meeting';
+
+module.exports = {
+  mode: 'development',
+  entry: ['./src/index.tsx'],
+  devtool: 'inline-source-map',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/
+      },
+    ]
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+    alias: {
+      react: path.resolve('./node_modules/react'),
+      'styled-components': path.resolve('./node_modules/styled-components'),
+      'react-dom': path.resolve('./node_modules/react-dom'),
+    },
+    fallback: {
+      fs: false,
+      tls: false,
+    }
+  },
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: `${app}-bundle.js`,
+    publicPath: '/',
+    libraryTarget: 'var',
+    library: `app_${app}`
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      inlineSource: '.(js|css)$',
+      template: __dirname + `/public/${app}.html`,
+      filename: __dirname + `/dist/${app}.html`,
+      inject: 'head'
+    }),
+    new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [new RegExp(`${app}`)]),
```

Comment:
We don't need this so you can remove it and remove the `react-dev-util` dependencies. This will save us some security issue :D

Reply:
Deleted.